app/models/badges/all_your_base_badge.rb

```diff
@@ -5,9 +5,11 @@ class AllYourBaseBadge < Badge
       'all-your-base',
       'Completed the "All Your Base" exercise'
 
-    def award_to?(user, exercise:)
-      return false unless exercise == 'all-your-base'
+    def self.worth_queuing?(context)
```

Comment:
```suggestion
    def self.worth_queuing?(exercise:)
```

I decided I liked this in the end. I actually think its much safer than using hash params.

Reply:
Great. I'll update